dist/adguard.json

```
@@ -60,6 +60,12 @@
             "url": "https://hockeyapp.net/",
             "companyId": null
         },
+        "google_trust_services": {
+            "name": "Google Trust Services",
+            "categoryId": 6,
```

```suggestion
            "categoryId": 5,
```

This should be `"5": "essential",`, not analytics